packages/block-logs-stream/tsconfig.json

```diff
@@ -9,6 +9,7 @@
     "isolatedModules": true,
     "esModuleInterop": true,
     "forceConsistentCasingInFileNames": true,
+    "skipLibCheck": true,
```

Comment:
what are the consequences of this?

Reply:
`"skipLibCheck": true` is necessary for using viem, and even in the viem repository, the same error occurs when removing the `"skipLibCheck": true`. I believe there isn't a good way to maintain `"skipLibCheck": false`, although I don't fully understand the exact cause of the error.

Skipping the check should be fine here because the declaration files are not utilized in the packages where I enabled the flag, making it relevant only for library files.